### PR TITLE
bmdca 0.8.12 (new formula)

### PR DESCRIPTION
**Pull request by YoshitakaMo**
- [x] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

This is a new formula of bmDCA (see https://github.com/ranganathanlab/bmDCA).

### CODE REVIEW

**Comment by YoshitakaMo** [comment]
Sorry, I accidentally deleted this PR. I hope you will reopen.

File: Formula/bmdca.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    system "./autogen.sh"
+    system "./configure", "--prefix=#{prefix}"
+
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    system "bmdca", "-h"
```
Comment (sjackman) [inline Formula/bmdca.rb:33]:
```suggestion
    assert_match "Usage", shell_output("#{bin}/bmdca -h")
```

**Review by sjackman — CHANGES_REQUESTED** [review]
One minor suggestion for the test block. Otherwise looks great. Thanks!

**Review by sjackman — APPROVED** [review]

File: Formula/pymol.rb [hunk 2]
```diff
@@ -70,7 +70,7 @@ def install
 
     ENV.append "CPPFLAGS", "-I#{Formula["freetype"].opt_include}"
 
-    # Note: openvr support is not included.
+    # openvr support is not included.
```
Comment (sjackman) [inline Formula/pymol.rb:73]:
```suggestion
    # Note: openvr support is not included.
```
Please revert this unrelated change.

**Review by sjackman — CHANGES_REQUESTED** [review]

File: Formula/pymol.rb (hunk 2 above)
Comment (YoshitakaMo) [reply Formula/pymol.rb]: For some reason, this change is necessary to pass the CI. Here is the message:

```
==> brew doctor

==> Running TapSyntax#run!
==> brew readall --aliases brewsci/bio
==> brew style brewsci/bio
==> FAILED
/home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/brewsci/homebrew-bio/Formula/pymol.rb:73:7: C: Annotation keywords like Note should be all upper case, followed by a colon, and a space, then a note describing the problem.
    # Note: openvr support is not included.
      ^^^^^^

305 files inspected, 1 offense detected, 1 offense auto-correctable
```

I've never encountered this message until today.

Comment (sjackman) [reply Formula/pymol.rb]: Me neither. That's a new one to me. I've pushed 8b30424cbe14bb2c089e20f2580e4d33b10fb696 to remove `Note:`.

Comment (sjackman) [reply Formula/pymol.rb]: You can rebase this PR onto `origin/develop` now.

Comment (YoshitakaMo) [reply Formula/pymol.rb]: Thank you. This PR is ready to merge now.

**Review by sjackman — APPROVED** [review]

**Comment by sjackman** [comment]
Merged. Thanks again for your contributions to Brewsci/bio, Yoshitaka! 🌟
